src/draive/splitters/basic.py

```diff
@@ -9,11 +9,11 @@ def split_text(
     text: str,
     part_size: int,
     count_size: Callable[[str], int],
-    separators: tuple[str, str] | str | None = None,
+    separators: Sequence[str] | str | None = None,
```

Comment:
I'd use `list` instead of the `Sequence` to narrow it down and to keep it simple. You could always pass and array or sth like that which implements `Sequence` interface whereas it's just a simple list of iterators.

Could you elaborate what was wrong with the `tuple` in first place?

Reply:
Which types? Who would pass separators as a numpy array? :3
Sometimes having less choice is better.

Reply:
yes, but I use this for backwards compatibility